README.md

```diff
@@ -102,7 +104,7 @@ Execute `npm run clean` to delete these tarballs when you no longer need them.
 ### Connecting to Postgres and Redis
 The Postgres and Redis services running on the Livepush device can be accessed with:
 ```
-$ psql -hpostgres.ly.fish.local -Udocker jellyfish -W (password = docker)
+$ psql -h postgres.ly.fish.local -U docker jellyfish -W (password = docker)
```

Comment:
I don't think we need spaces here.

Reply:
Indeed it works without spaces. I will remove these changes from PR.